samples/wifi/throughput/README.rst

```diff
@@ -0,0 +1,308 @@
+   +==============+================================+============================================================+
+   | UDP          | zperf udp download <dest port> | iperf -c <ipaddress> -i 1 -t <duration> -u -b <baud rate>  |
+   +--------------+--------------------------------+------------------------------------------------------------+
+   | TCP          | zperf tcp download <dest port> | iperf -c <ipaddress> -i 1 -t <duration> -b <baud rate>     |
+   +--------------+--------------------------------+------------------------------------------------------------+
+
+
+   The sample shows the following output:
+
+   .. code-block:: console
+
+      $ zperf udp download 5001
+      UDP server started on port 5001
+      <inf> net_zperf: Binding to 192.168.1.13
+      <inf> net_zperf: Binding to ::
+      <inf> net_zperf: Listening on port 5001
+      New session started.
+      End of session!
+      duration:              10.02 s
+      received packets:      10694
+      nb packets lost:       33884
+      nb packets outorder:   0
+      jitter:                        5.20 ms
+      rate:                  11.96 Mbps
+
+#. Stop the ``zperf`` udp/tcp download using the following command:
+
+   .. code-block:: console
+
+      zperf <udp/tcp> download stop
+
+   The sample shows the following output:
+
+   .. code-block:: console
+
+      $ zperf udp download stop
+      UDP server stopped
+
+
+Results
+=======
+
+The following table collects a summary of results obtained when throughput tests are run for different profiles.
+
+
++------------------------+-----------+-----------+-----------+-----------+
+| Profile                | UDP TX    | UDP RX    | TCP TX    | TCP RX    |
+|                        |           |           |           |           |
++========================+===========+===========+===========+===========+
+| IoT devices            | 5.08 Mbps | 5.43 Mbps | 5.25 Mbps | 3.35Mbps  |
++------------------------+-----------+-----------+-----------+-----------+
+| Memory-optimized       | 5.1 Mbps  | 113 Kbps  | 644 Kbps  | 1.86 Mbps |
++------------------------+-----------+-----------+-----------+-----------+
+| High performance       | 26.2 Mbps | 13.08 Mbps| 14.1 Mbps | 7.76 Mbps |
++------------------------+-----------+-----------+-----------+-----------+
+| TX prioritized         | 26.3 Mbps | 12.20 Mbps| 10.5 Mbps | 4.5 Mbps  |
++------------------------+-----------+-----------+-----------+-----------+
+| RX prioritized         | 9.34 Mbps | 13.46 Mbps| 5.48 Mbps | 7.92 Mbps |
++------------------------+-----------+-----------+-----------+-----------+
+
+.. note::
+   The measured throughputs, as shown in the table above, are based on tests conducted using the nRF7002 DK.
+   The results represent the best throughput, averaged over three iterations, and were obtained with a good RSSI signal in a clean environment.
```

Comment:
This is fine. Ideally you could add a recommendation for how users can achieve a "clean" environment. I.e. what makes the environment less clean? Would it help running this in a chamber, etc.

Reply:
Hi @ioannisg ,
This setup is placed in RF Chamber. Can I write it this way, 
"The results represent the best throughput, averaged over three iterations, and were obtained with a good RSSI signal in a clean environment (RF Chamber)."